OpenOversight/app/templates/submit_image.html

```diff
@@ -84,7 +84,7 @@ <h3>Drop images here to submit photos of officers:</h3>
 
 <div class="container">
    <h3>High Security Submissions</h3>
-   <p>We do not log unique identifying information of visitors to our website, but if you have privacy concerns in submitting photographs, we recommend using <a href="https://www.torproject.org/projects/torbrowser.html.en">Tor Browser.</a></p>
+   <p>We do not log unique identifying information of visitors to our website, but if you have privacy concerns in submitting photographs, we recommend using <a href="https://www.torproject.org/projects/torbrowser.html.en" rel="noopener noreferrer" target="_blank">Tor Browser.</a></p>
```

Comment:
It might be good to add a note here about how we scrub EXIF data!

Reply:
Yup! Done in 6746aa2